src/test/java/com/microsoft/jenkins/artifactmanager/AzureArtifactConfigTest.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.jenkins.artifactmanager;
+
+import hudson.util.FormValidation;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.mockito.ArgumentMatchers.any;
+
+public class AzureArtifactConfigTest {
+    AzureArtifactConfig.DescriptorImpl descriptor;
+
+    @Before
+    public void init() {
+        descriptor = Mockito.mock(AzureArtifactConfig.DescriptorImpl.class);
+        Mockito.when(descriptor.doCheckContainer(any())).thenCallRealMethod();
+    }
+
+
+    @Test
+    public void testContainerName() throws Exception {
```

Comment:
normally you would do each scenario as a different test case, but thanks for adding the unit test good test coverage there